tests/ftests/048-sudo-systemd_create_scope.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# Create a systemd scope
+#
+# Copyright (c) 2022 Oracle and/or its affiliates.
+# Author: Tom Hromatka <tom.hromatka@oracle.com>
+#
+
+from cgroup import Cgroup as CgroupCli
+from cgroup import CgroupVersion
+from libcgroup import Cgroup
+from systemd import Systemd
+from run import Run
+import ftests
+import consts
+import time
```

Comment:
The import time is not used.

Reply:
Testcase No `048` is already taken by `048-pybindings-get_cgroup_mode.py`.

Reply:
Yep, I need to rebase now that the setup_mode was added to main

Reply:
Done